steps/compliance/dwp-issuing-office-other/DWPIssuingOfficeEsa.js

```diff
@@ -14,6 +14,9 @@ class DWPIssuingOfficeEsa extends SaveToDraftStore {
 
   static selectify(ar) {
     return ar.map(el => {
+      if (el === 'Recovery from Estates') {
```

Comment:
I think we should check with the code instead of value, as 'Recovery from Estates' will be different in welsh